tests/sources/test_jira.py

```diff
@@ -155,6 +155,110 @@
+        "accountType": "atlassian",
+        "displayName": "user1",
+        "active": True,
+    },
+)
+
+MOCK_SECURITY_LEVEL = {
+    "id": "10000",
+    "key": "TP-1",
+    "fields": {"security": {"id": "level-1", "name": "issue-level"}},
+}
+
+MOCK_SECURITY_LEVEL_MEMBERS = {
+    "maxResults": 50,
+    "startAt": 0,
+    "total": 2,
+    "isLast": True,
+    "values": [
+        {
+            "id": "level-1",
+            "issueSecurityLevelId": "10000",
+            "issueSecuritySchemeId": "10002",
+            "holder": {
+                "type": "user",
+                "parameter": "user-1",
+                "value": "user-1",
+                "user": {
+                    "self": "url",
+                    "accountId": "user-1",
+                    "emailAddress": "test.user@gmail.com",
+                    "displayName": "Test User",
+                    "active": True,
+                    "timeZone": "Asia/Calcutta",
+                    "accountType": "atlassian",
+                },
+                "expand": "user",
+            },
+        },
+        {
+            "id": "level-2",
+            "issueSecurityLevelId": "10000",
+            "issueSecuritySchemeId": "10002",
+            "holder": {
+                "type": "group",
+                "parameter": "Project-admins",
+                "value": "d614bbb8-703d-4d41-a51a-0c3cfcef9318",
+                "group": {
+                    "name": "Project-admins",
+                    "groupId": "d614bbb8-703d-4d41-a51a-0c3cfcef9318",
+                    "self": "self",
+                },
+                "expand": "group",
+            },
+        },
+        {
+            "id": "level-3",
+            "issueSecurityLevelId": "10000",
+            "issueSecuritySchemeId": "10002",
+            "holder": {
+                "type": "projectRole",
+                "parameter": "10002",
+                "value": "10002",
+                "projectRole": {
+                    "self": "self",
+                    "name": "Administrator",
+                    "id": 10002,
+                },
+                "expand": "projectRole",
+            },
+        },
+    ],
+}
+
+MOCK_PROJECT_ROLE_MEMBERS = {
+    "self": "self",
+    "name": "Administrators",
+    "id": 10002,
+    "actors": [
+        {
+            "id": 11542,
+            "displayName": "Test User",
+            "type": "atlassian-user-role-actor",
+            "actorUser": {"accountId": "5ff5815e34847e0069fedee3"},
+        },
+        {
+            "id": 11543,
+            "displayName": "Project-admins",
+            "type": "atlassian-group-role-actor",
+            "name": "Project-admins",
+            "actorGroup": {
+                "name": "Project-admins",
+                "displayName": "Project-admins",
+                "groupId": "d614bbb8-703d-4d41-a51a-0c3cfcef9318",
+            },
+        },
+    ],
+}
+
+ACCESS_CONTROL = "_allow_access_control"
+
 
 class mock_ssl:
```

Comment:
```suggestion
class MockSsl:
```

nit: class names should be PascalCase.